src/helm/benchmark/scenarios/commonsense_scenario.py

```diff
@@ -236,15 +284,13 @@ def load_dataset(self, output_path: str) -> List[List[str]]:
         return data
 
     def get_instances(self, output_path: str) -> List[Instance]:
+        if self.dataset == "hellaswag":
+            return HellaswagScenario().get_instances(output_path)
+        if self.dataset == "openbookqa":
+            return OpenBookQA().get_instances(output_path)
```

Comment:
I'd suggest doing the scenario switching in `get_commonsense_spec()` [here](https://github.com/stanford-crfm/helm/blob/8d4432120f1de374851529c831c8fab2ef28e55d/src/helm/benchmark/run_specs.py#L980-L984) i.e. something like:

```python
def get_commonsense_spec(dataset: str, method: str) -> RunSpec:
    if dataset == hellaswag:
        scenario_spec = ScenarioSpec(
            class_name="helm.benchmark.scenarios.commonsense_scenario.HellaSwagScenario",
            args={},
        )
```

Then we don't need to spawn a scenario within a scenario, and also it makes sure that the scenario name gets recorded correctly.

Reply:
Done. One thing to note is this will move where the data is written. Previously it was in `scenarios/commonsense/data/hellaswag/`, and now it will be in `scenarios/hellaswag/data/`.